sympy/solvers/solveset.py

```diff
@@ -1536,7 +1537,7 @@ def _invert_modular(modterm, rhs, n, symbol):
         # Check for complex arguments
         return modterm, rhs
 
-    if abs(rhs) >= abs(m):
+    if ask(abs(rhs) >= abs(m)):
```

Comment:
This should probably be something like:
```
if (abs(rhs) >= abs(m)) is S.true:
```
See:
https://docs.sympy.org/latest/guides/booleans.html